src/main/webapp/resources/js/pages/analysis/components/Analysis.jsx

```diff
@@ -41,6 +42,19 @@ const AnalysisProvenance = lazy(() => import("./AnalysisProvenance"));
 export default function Analysis() {
   const { analysisContext } = useContext(AnalysisContext);
   const DEFAULT_URL = `/analysis/${analysisContext.analysis.identifier}`;
+  const [treeDefault, setTreeDefault] = useState(null);
+
+  useEffect(() => {
+    if((analysisType === "PHYLOGENOMICS" || analysisType === "MLST_MENTALIST") && analysisContext.isCompleted) {
+      getNewickTree(analysisContext.analysis.identifier).then(data => {
```

Comment:
If the tree doesn't exist the page just crashes.  There should be some error handler here to gracefully die if the tree file isn't there.

Reply:
Fixed :)